### DISCUSSION TITLE
Crash in editor

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#20734] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

It just crashed in editor upon seek? or randomly i am unsure

### Screenshots or videos

N/A

### Version

2022.1008.2

### Logs

[logs.zip](https://github.com/ppy/osu/files/9770276/logs.zip)

**Comment by peppy** [comment, marked as answer]
There's no crash in your logs file. This usually means your system is failing. You can check event viewer, there may be a GPU level crash or similar.

**Reply by Axilotl17** [reply]
Ill take your word for it and check event viewer if it happens again.
